Logboek wordt niet geladen / blijft leeg

Zelfs na een nieuwe installatie wordt het logboek van de addon niet geladen: 
[https://i.imgur.com/0ggcrQ2.png](url)
Vernieuwen of downloaden van de logs heeft geen effect.

DAO 2025.1.3
RPI 4B
Core 2025.1.4
Supervisor 2024.12.3

Via Systeem>Logboeken>Day Ahead Optimizer wordt het logboek wel getoond:


Traceback (most recent call last):
  File "/root/dao/prog/da_scheduler.py", line 65, in <module>
    main()
  File "/root/dao/prog/da_scheduler.py", line 60, in main
    da_sched = DaScheduler("../data/options.json")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/dao/prog/da_scheduler.py", line 10, in __init__
    super().__init__(file_name)
  File "/root/dao/prog/da_base.py", line 90, in __init__
    super().__init__(hassurl=self.hassurl, token=self.hasstoken)
  File "/root/dao/venv/day_ahead/lib/python3.11/site-packages/hassapi/client/base.py", line 36, in __init__
    self._assert_api_running()
  File "/root/dao/venv/day_ahead/lib/python3.11/site-packages/hassapi/client/base.py", line 41, in _assert_api_running
    raise ClientError("Home Assistant API is not running.")
hassapi.exceptions.ClientError: Home Assistant API is not running.
[2025-02-02 11:35:14 +0100] [14] [ERROR] Can't connect to ('0.0.0.0', 5000)
=> directory dao_data exist
=> /root/dao/data doesn't exist, made
=> /root/dao/webserver/app/static/data exist
Setting up watches.
Watches established.
[2025-02-02 11:46:23 +0100] [12] [INFO] Starting gunicorn 23.0.0
[2025-02-02 11:46:23 +0100] [12] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 13] Permission denied
[2025-02-02 11:46:24 +0100] [12] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 13] Permission denied
[2025-02-02 11:46:25 +0100] [12] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 13] Permission denied
[2025-02-02 11:46:26 +0100] [12] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 13] Permission denied
[2025-02-02 11:46:27 +0100] [12] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 13] Permission denied
[2025-02-02 11:46:28 +0100] [12] [ERROR] Can't connect to ('0.0.0.0', 5000)
Traceback (most recent call last):
  File "/root/dao/prog/da_scheduler.py", line 65, in <module>
    main()
  File "/root/dao/prog/da_scheduler.py", line 60, in main
    da_sched = DaScheduler("../data/options.json")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/dao/prog/da_scheduler.py", line 10, in __init__
    super().__init__(file_name)
  File "/root/dao/prog/da_base.py", line 90, in __init__
    super().__init__(hassurl=self.hassurl, token=self.hasstoken)
  File "/root/dao/venv/day_ahead/lib/python3.11/site-packages/hassapi/client/base.py", line 36, in __init__
    self._assert_api_running()
  File "/root/dao/venv/day_ahead/lib/python3.11/site-packages/hassapi/client/base.py", line 41, in _assert_api_running
    raise ClientError("Home Assistant API is not running.")
hassapi.exceptions.ClientError: Home Assistant API is not running.

Ik blijf deze error krijgen:
https://i.imgur.com/gOomg1C.png 

En dit in het logboek:
=> directory dao_data exist
=> /root/dao/data doesn't exist, made
=> /root/dao/webserver/app/static/data exist
Setting up watches.
Watches established.
[2025-02-03 09:32:58 +0100] [12] [INFO] Starting gunicorn 23.0.0
[2025-02-03 09:32:58 +0100] [12] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 13] Permission denied
[2025-02-03 09:32:59 +0100] [12] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 13] Permission denied
[2025-02-03 09:33:00 +0100] [12] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 13] Permission denied
[2025-02-03 09:33:01 +0100] [12] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 13] Permission denied
[2025-02-03 09:33:02 +0100] [12] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 13] Permission denied
Traceback (most recent call last):
  File "/root/dao/prog/da_scheduler.py", line 65, in <module>
    main()
  File "/root/dao/prog/da_scheduler.py", line 60, in main
    da_sched = DaScheduler("../data/options.json")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/dao/prog/da_scheduler.py", line 10, in __init__
    super().__init__(file_name)
  File "/root/dao/prog/da_base.py", line 90, in __init__
    super().__init__(hassurl=self.hassurl, token=self.hasstoken)
  File "/root/dao/venv/day_ahead/lib/python3.11/site-packages/hassapi/client/base.py", line 36, in __init__
    self._assert_api_running()
  File "/root/dao/venv/day_ahead/lib/python3.11/site-packages/hassapi/client/base.py", line 41, in _assert_api_running
    raise ClientError("Home Assistant API is not running.")
hassapi.exceptions.ClientError: Home Assistant API is not running.
[2025-02-03 09:33:03 +0100] [12] [ERROR] Can't connect to ('0.0.0.0', 5000)

Aanpassen van de poort naar bijv. 5025 heeft blijkbaar geen effect, althans in de log staat nog steeds 5000.

Nog geen verschil:

=> directory dao_data exist
=> /root/dao/data doesn't exist, made
=> /root/dao/webserver/app/static/data exist
Setting up watches.
Watches established.
[2025-02-03 23:07:42 +0100] [12] [INFO] Starting gunicorn 23.0.0
[2025-02-03 23:07:42 +0100] [12] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 13] Permission denied
[2025-02-03 23:07:43 +0100] [12] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 13] Permission denied
[2025-02-03 23:07:44 +0100] [12] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 13] Permission denied
[2025-02-03 23:07:45 +0100] [12] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 13] Permission denied
[2025-02-03 23:07:46 +0100] [12] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 13] Permission denied
Traceback (most recent call last):
  File "/root/dao/prog/da_scheduler.py", line 65, in <module>
    main()
  File "/root/dao/prog/da_scheduler.py", line 60, in main
    da_sched = DaScheduler("../data/options.json")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/dao/prog/da_scheduler.py", line 10, in __init__
    super().__init__(file_name)
  File "/root/dao/prog/da_base.py", line 90, in __init__
    super().__init__(hassurl=self.hassurl, token=self.hasstoken)
  File "/root/dao/venv/day_ahead/lib/python3.11/site-packages/hassapi/client/base.py", line 36, in __init__
    self._assert_api_running()
  File "/root/dao/venv/day_ahead/lib/python3.11/site-packages/hassapi/client/base.py", line 41, in _assert_api_running
    raise ClientError("Home Assistant API is not running.")
hassapi.exceptions.ClientError: Home Assistant API is not running.
[2025-02-03 23:07:47 +0100] [12] [ERROR] Can't connect to ('0.0.0.0', 5000)

--- a/dao/apparmor.txt
+++ b/dao/apparmor.txt
@@ -5,6 +5,8 @@ profile day_ahead_opt flags=(attach_disconnected,mediate_deleted) {
   #include <abstractions/base>
 
   # Capabilities
+  network,
+  deny network raw,
   file,
   signal (send) set=(kill,term,int,hup,cont),
 

